src/runtime/config/configuration-qemu.toml.in

```diff
@@ -26,13 +26,27 @@ machine_type = "@MACHINETYPE@"
 #
 # Known limitations:
 # * Does not work by design:
-#   - CPU Hotplug 
+#   - CPU Hotplug
 #   - Memory Hotplug
 #   - NVDIMM devices
 #
 # Default false
 # confidential_guest = true
 
+# Enable pre-attestation AMD SEV and SEV-ES confidential guests.
+# Applies only if confidential_guest is true.
+# (default: false)
+#guest_pre_attestation = true
+#
+# Guest owner proxy that handles remote attestation
+#guest_pre_attestation_proxy="localhost:50051"
+#
+# Keyset ID for injected secrets
+#guest_pre_attestation_keyset="KEYSET-1"
+#
+# SEV guest policy
+#sev_guest_policy=0
```

Comment:
One thing that I wonder here is whether we should have those generic / opaque to whether using SEV / TDX, but this is not a blocker here ... but if we're able to consolidate this before getting it merged into`main`that would be desirable.

Reply:
Yes, this is worth a discussion. One thing to consider is that, compared to SNP and TDX, only SEV does pre-attestation and therefore requires the handshake to be done by the shim. However, in all cases, we may need to communicate to the guest the remote proxy, policy, and keyset via the configuration or pod annotations.  

cc @fitzthum @sameo